index.html

```diff
@@ -4,22 +4,67 @@
 <head>
   <link rel="stylesheet" href="css/styles.css">
   <link rel="icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Merriweather:wght@300&family=Montserrat:wght@300&family=Sacramento&display=swap" rel="stylesheet">
```

Comment:
Use Atom beautify to align your code for easier readability

Reply:
this has been fixed